src/NuGet.Core/NuGet.CommandLine.XPlat/xlf/Strings.de.xlf

```diff
@@ -893,69 +893,69 @@ Weitere Informationen finden Sie unter: https://docs.nuget.org/docs/reference/co
-        <target state="new">Package '{0}' ({1}) failed to download.</target>
+        <target state="translated">Fehler beim Herunterladen des Pakets {0} ({1}).</target>
         <note>0 - package id
 1 - package version</note>
       </trans-unit>
       <trans-unit id="PackageDownloadCommand_LatestVersion">
         <source>latest version</source>
-        <target state="new">latest version</target>
+        <target state="translated">aktuellste Version</target>
         <note />
       </trans-unit>
       <trans-unit id="PackageDownloadCommand_OutputDirectoryDescription">
         <source>Directory where the package will be placed. Defaults to the current working directory.</source>
-        <target state="new">Directory where the package will be placed. Defaults to the current working directory.</target>
+        <target state="translated">Verzeichnis, in dem das Paket abgelegt wird. Standardmäßig ist das das aktuelle Arbeitsverzeichnis.</target>
         <note />
       </trans-unit>
       <trans-unit id="PackageDownloadCommand_PackageIdDescription">
         <source>Package identifier (e.g. 'Newtonsoft.Json').</source>
-        <target state="new">Package identifier (e.g. 'Newtonsoft.Json').</target>
+        <target state="translated">Paketbezeichner (z. B. „Newtonsoft.Json“).</target>
         <note />
       </trans-unit>
       <trans-unit id="PackageDownloadCommand_PackageSourceMapping_NoSuchSource">
         <source>The mapped source '{0}' for package '{1}' was not found among the configured sources.</source>
-        <target state="new">The mapped source '{0}' for package '{1}' was not found among the configured sources.</target>
+        <target state="translated">Die zugeordnete Quelle {0} für das Paket {1} wurde unter den konfigurierten Quellen nicht gefunden.</target>
         <note>0 - package source name
 1 - package name</note>
       </trans-unit>
       <trans-unit id="PackageDownloadCommand_SourcesDescription">
         <source>Specifies one or more NuGet package sources to use.</source>
-        <target state="new">Specifies one or more NuGet package sources to use.</target>
+        <target state="translated">Gibt eine oder mehrere NuGet-Paketquellen an, die verwendet werden sollen.</target>
         <note />
       </trans-unit>
       <trans-unit id="PackageDownloadCommand_Starting">
         <source>Downloading package {0}, version {1}.</source>
-        <target state="new">Downloading package {0}, version {1}.</target>
+        <target state="translated">Paket {0}wird heruntergeladen, Version {1}.</target>
         <note>0- package id
 1- package version</note>
       </trans-unit>
       <trans-unit id="PackageDownloadCommand_Succeeded">
         <source>Package '{0}' ({1}) successfully downloaded to '{2}'.</source>
-        <target state="new">Package '{0}' ({1}) successfully downloaded to '{2}'.</target>
+        <target state="translated">Das Paket {0} ({1}) wurde erfolgreich in {2} heruntergeladen.</target>
```

Comment:
It could be culturally significant, but I noticed the quotes were removed here.

Reply:
I think so, other strings in this file also removed the ' ', and other languages use other things, since it's not new behavior I think this is correct